helper.js

```diff
@@ -0,0 +1,15 @@
+const configureIframeCoverage = (config) => {
```

Comment:
This file needs to be added to the "files" array in the `package.json` to be included in the npm package.